src/spec-lang/tc/typecheck.ts

```diff
@@ -1470,6 +1475,34 @@ export function tcMemberAccess(expr: SMemberAccess, ctx: STypingCtx, typeEnv: Ty
         }
     }
 
+    // User defined value types wrap() and unwrap()
+    if (
+        baseT instanceof TypeNameType &&
+        baseT.type instanceof UserDefinedType &&
+        baseT.type.definition instanceof UserDefinedValueTypeDefinition
+    ) {
+        const userDefValType = baseT.type.definition;
+        const underlyingType = userDefValType.underlyingType;
+
+        if (expr.member === "wrap") {
+            return new FunctionType(
+                "wrap",
+                [typeNameToTypeNode(underlyingType)],
+                [baseT.type],
+                FunctionVisibility.Default,
+                FunctionStateMutability.Pure
+            );
+        } else if (expr.member === "unwrap") {
```

Comment:
`else if` could be an `if` fron new line - would be a bit easier to read/scan the code flow

Reply:
Fixed in 9555f70.